docs/rules/lines-around-comment.md

```diff
@@ -48,123 +24,115 @@ Empty lines are also not required at the beginning or end of a file.
+* `"beforeBlockComment": true` (default) requires an empty line before block comments
+* `"beforeBlockComment": false` disallows an empty line before block comments
+* `"afterBlockComment": true` requires an empty line after block comments
+* `"beforeLineComment": true` requires an empty line before line comments
+* `"afterLineComment": true` requires an empty line after line comments
+* `"allowBlockStart": true` allows comments to appear at the start of block statements
+* `"allowBlockEnd": true` allows comments to appear at the end of block statements
+* `"allowObjectStart": true` allows comments to appear at the start of object literals
+* `"allowObjectEnd": true` allows comments to appear at the end of object literals
+* `"allowArrayStart": true` allows comments to appear at the start of arrays
+* `"allowArrayEnd": true` allows comments to appear at the end of arrays
 
-* `beforeBlockComment` (enabled by default)
-* `afterBlockComment`
+### beforeBlockComment
 
-2 options for line comments:
+Examples of **incorrect** code for this rule with the default `{ "beforeBlockComment": true }` option:
 
-* `beforeLineComment`
-* `afterLineComment`
+```js
+/*eslint lines-around-comment: ["error", { "beforeBlockComment": true }]*/
 
-6 options for exceptions:
+var night = "long";
+/* what a great and wonderful day */
+var day = "great"
+```
 
-* `allowBlockStart`
-* `allowBlockEnd`
-* `allowObjectStart`
-* `allowObjectEnd`
-* `allowArrayStart`
-* `allowArrayEnd`
+Examples of **correct** code for this rule with the default `{ "beforeBlockComment": true }` option:
 
-Any combination of these rules may be applied at the same time.
+```js
+/*eslint lines-around-comment: ["error", { "beforeBlockComment": true }]*/
 
+var night = "long";
 
-```json
-{
-    "lines-around-comment": ["error", { "beforeBlockComment": true, "beforeLineComment": true }]
-}
+/* what a great and wonderful day */
+var day = "great"
 ```
 
-When set to `false` the option is simply ignored.
-
-
-### Block Comments
-
-Block comments are any comment that start with `/*` and need not extend to multiple lines.
+### afterBlockComment
 
-With both `beforeBlockComment` and `afterBlockComment` set to `true` the following code
-would not warn:
+Examples of **incorrect** code for this rule with the `{ "afterBlockComment": true }` option:
 
 ```js
-/*eslint lines-around-comment: ["error", { "beforeBlockComment": true, "afterBlockComment": true }]*/
+/*eslint lines-around-comment: ["error", { "afterBlockComment": true }]*/
 
 var night = "long";
-
 /* what a great and wonderful day */
-
 var day = "great"
 ```
 
-This however would provide 2 warnings:
+Examples of **correct** code for this rule with the `{ "afterBlockComment": true }` option:
 
 ```js
-/*eslint lines-around-comment: ["error", { "beforeBlockComment": true, "afterBlockComment": true }]*/
+/*eslint lines-around-comment: ["error", { "afterBlockComment": true }]*/
 
 var night = "long";
 /* what a great and wonderful day */
```

Comment:
We do need the empty line preceding this block comment because of the default beforeBlockComment option.